### ISSUE TITLE
Is this project supports .js file with jsdoc?

### ISSUE THREAD

**Issue by easyfrog** [FlorianRappl/dets#41]
I use js & jsdoc in my project. But when I use dets to extract types to a single d.ts. the targert d.ts file is empty.

```bash
dets --files src/projects/Navigator.js --out dist/js/Navigator.d.ts   
```

`Navigator.d.ts` is empty.

**Comment by FlorianRappl** [comment]
I am not sure how much effort it would be to also support jsdoc; I guess it should be managable as `tsc` also reads this information in. I'd need to check this out.

**Comment by FlorianRappl** [comment]
Sorry I did not manage to look into this yet. I'll try to have it on my list for next week.

I'm scheduling this for the 0.15 release.

**Comment by FlorianRappl** [comment]
Available in 0.16. Should work if you use the command like:

```sh
dets src/projects/Navigator.js --out dist/js/Navigator.d.ts   
```

(No `--files` being used; the positional sets `files` and `types`: `types` was missing beforehand)
